CHANGELOG.next.toml

```diff
@@ -9,4 +9,10 @@
 # message = "Fix typos in module documentation for generated crates"
 # references = ["smithy-rs#920"]
 # meta = { "breaking" = false, "tada" = false, "bug" = false, "target" = "client | server | all"}
-# author = "rcoh"
\ No newline at end of file
+# author = "rcoh"
+
+[[aws-sdk-rust]]
+message = "Add support for `[sso-session]` in AWS config file for AWS Identity Center SSO credentials."
+references = ["aws-sdk-rust#703", "smithy-rs#3379"]
```

Comment:
would probably be helpful to include an example profile in the message. Since this doesn't support loading tokens yet (right?) it should probably call that out and link to the tracking issue.

Reply:
Opted not to include an example since the recommended approach for SSO is to use the AWS CLI to configure it (via `aws configure sso` and `aws sso login`). Added more information about the lack of Builder ID support.